src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -580,7 +580,7 @@ private function buildTableRows(array $rows)
             }
         }
 
-        return new TableRows(function () use ($rows, $unmergedRows) {
+        return new TableRows(function () use ($rows, $unmergedRows): \Traversable {
```

Comment:
`iterable` like in the other places?

Reply:
The `TableRows` class requires a callable that returns a real `\Traversable`. This is why I did not use `iterable` as return type here.